frontend/public/components/configmap-and-secret-data.jsx

```diff
@@ -1,17 +1,38 @@
+import * as _ from 'lodash-es';
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
-import { Heading } from './utils';
+import { Heading, CopyToClipboard } from './utils';
 
-export const ConfigMapData = ({data}) => {
+export const MaskedData = () => {
+  return <React.Fragment>
+    <span className="sr-only">value hidden</span>
+    <span aria-hidden="true">&bull;&bull;&bull;&bull;&bull;</span>
+  </React.Fragment>;
+}
```

Comment:
Don't export items we aren't using elsewhere. Suggest just calling it `Masked`. This could be

```jsx
const Masked = <React.Fragment>
  <span className="sr-only">value hidden</span>
  <span aria-hidden="true">&bull;&bull;&bull;&bull;&bull;</span>
</React.Fragement>
```